src/main/java/com/auth0/client/mgmt/GuardianEntity.java

```diff
@@ -23,6 +23,7 @@
 
     /**
      * Create a new Guardian Enrollment Ticket. A token with scope create:guardian_enrollment_tickets is needed.
+     * <p/>
```

Comment:
self closing paragraph `<p/>` produces a Javadoc lint issue. If you explicitly need it then wrap the paragraph with `<p> text </p>`